week-3/Homework/mandatory/3-space-colonies.js

```diff
@@ -8,10 +8,20 @@
   NOTE: don't include any element that is not a "family".
 */
 
-function colonisers() {
-
+function colonisers(voyagers) {
+  let result = voyagers.filter(Element => Element.indexOf("family") >= 0 && Element.charAt(0) === "A");
```

Comment:
This is correct, great work!

However there is a new string method that is a bit nicer than `indexOf` for checking whether a string contains another string: [`.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes). How would this improve your code?